packages/services/src/kernel/default.ts

```diff
@@ -1375,26 +1354,21 @@ export class KernelConnection implements Kernel.IKernelConnection {
           // Updating the status is synchronous, and we call no async user code
           const executionState = (msg as KernelMessage.IStatusMsg).content
             .execution_state;
-          if (executionState === 'autorestarting') {
+          if (executionState === 'restarting') {
```

Comment:
The notebook server emits a "restarting" status when there is an autorestart, so we must listen for "restarting". There is no "autorestarting" status emitted from the notebook server.